docs/source/data-model.rst

```diff
@@ -1,18 +1,39 @@
-**********
-Data Model
-**********
+********************************
+Data Model: Documents and Events
```

Comment:
I think of Events as one of the kinds of Documents.

Reply:
I have had some confusion with the event/Event terminology. In my mind there are event-based execution model "events" and then there are "Event Documents". I try not to think of the documents as "events" but as "Events". I think you are right, this heading should go back to just "Data Model".

Reply:
So, let's be clearer.  Instead of starting with the next paragraph, start with something more direct.

```
In data acquisition, an *event* is a set of readings coordinated at a specific time.  A primary design goal of bluesky is to enable better research by recording rich metadata alongside the event data for use in later analysis.  Bluesky encapsulates all information from each event in a *Document*.  Other types of Document describe the events associated with an experimental run.
```